testsuite/pytests/test_helper_functions.py

```diff
@@ -25,6 +25,11 @@
 
 class TestHelperFunctions(unittest.TestCase):
 
+    def test_load_help(self):
+        help_text = nest.hl_api.load_help('ac_generator')
```

Comment:
`hl_api` is gone and the `load_help` function is merely meant as a helper, so I'm not sure we actually should test it in this way.